.packit.yaml

```diff
@@ -0,0 +1,24 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+upstream_package_name: aardvark-dns
+downstream_package_name: aardvark-dns
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      - fedora-all-aarch64
+      - fedora-all-i386
+      - fedora-all-ppc64le
+      - fedora-all-s390x
+      - fedora-all-x86_64
+    srpm_build_deps:
+      - cargo
+      - make
+      - openssl-devel
+    actions:
+      post-upstream-clone:
+        - "curl -O https://src.fedoraproject.org/rpms/aardvark-dns/raw/rawhide/f/aardvark-dns.spec"
```

Comment:
Does this mean your downstream fedora spec file changes can break upstream CI?
I mean I agree that it is good so we can fix packaging bugs before we release but this is something we should be aware of.

Reply:
I *think* we can use `aardvark-dns.spec.rpkg` from upstream instead, but gotta verify. That would mean no dependency on fedora dist-git.

Reply:
hmm, doesn't work yet with spec.rpkg files. I'll check with the packit people. Not depending on fedora dist-git would be super convenient.

Reply:
@Luap99 now reuses the spec.rpkg file, so we're not dependent on official fedora package anymore.